sidebars.js

```diff
@@ -294,29 +294,47 @@ module.exports = {
           label: "Zowe CLI",
           link: {type:"doc", id:"user-guide/user-roadmap-zowe-cli"},
           items: [
-            "user-guide/systemrequirements-cli",
             "user-guide/cli-install-cli-checklist",
+            {
+              type: "category",
+              label: "Zowe CLI software requirements",
+              link: {type:"doc", id:"user-guide/systemrequirements-cli"},
+              items: [
+                "user-guide/cli-swreqplugins",
+              ],
+              },
+            "user-guide/cli-install-configure-install-online-registry-proxy",
+            "user-guide/cli-install-configure-zosmf",
+            "user-guide/cli-install-configure-zosmf-security",
           {
           type: "category",
-          label: "Installing Zowe CLI",
+          label: "Installing Zowe CLI and Zowe CLI plug-ins",
           link: {type:"doc", id:"user-guide/cli-installcli"},
           items: [
             "user-guide/cli-configure-scs-on-headless-linux-os",
             "user-guide/cli-configure-cli-on-os-where-scs-unavailable",
-            "user-guide/cli-install-cli-nodejs-windows",
           ],
           },
-            "user-guide/install-cli-via-proxy",
             "user-guide/cli-updatingcli",
-            "user-guide/cli-uninstall",   
             {
               type: "category",
-              label: "Advanced Zowe CLI configuration",
+              label: "Configuring Zowe CLI",
               items: [
                 "user-guide/cli-configuringcli-ev",
                 "user-guide/cli-configuringcli-evfile",
+                "user-guide/cli-using-initializing-team-configuration",
+                {
+                  type: "category",
+                  label: "Configuring daemon mode",
+                  link: {type:"doc", id:"user-guide/cli-using-using-daemon-mode"},
+                  items: [
+                    "user-guide/cli-configure-daemon-on-zlinux-os",
+                  ],
+                },
               ],
             }, 
+            "user-guide/cli-uninstall",
+            "user-guide/cli-install-verify-your-installation", 
```

Comment:
Would it make sense for this to be moved up, perhaps to just above `user-guide/cli-updatingcli`, or right after the `Configuring Zowe CLI` section? This doc seems out of place, instructing people how to verify their installation is correct right after the uninstalling section.

Reply:
Will think on this. Some updating processes have users uninstalling cli and plug-ins in order to re-installed a newer version of cli, plug-ins. That's why validation is listed after uninstalling. I think after all the V3 content is added, I'll think more seriously about topic order in the sidebar.